sdc-plugin/tests/counter/counter.golden.txt

```diff
@@ -1 +1,2 @@
-{$auto$clkbufmap.cc:247:execute$1918} {$auto$clkbufmap.cc:247:execute$1920} clk clk2 clk_int_1 ibuf_proxy_out middle_inst_1.clk_int middle_inst_4.clk
+clk clk2 clk_int_1
```

Comment:
This still shows the `get_clocks` and `get_clocks -include_generated_clocks` as the same?  Is there a golden example that shows `-include_generated_clocks` doing the right thing?

Reply:
I enhanced the `get_clocks` example which shows the difference.